File: RepoTransBench/source_projects/Java/AllenDowney_ThinkJavaCode/src/test/java/ap01/SeriesFibonacciEdgePublicTest.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

public class SeriesFibonacciEdgePublicTest {

    @Test
    public void testFibonacciLow() {
        // Test new value n for Fibonacci sequence
        assertEquals(13, ap01.Series.fibonacci(7));
        assertEquals(21, ap01.Series.fibonacci(8));
    }

    @Test
    public void testFibonacciHighDifferentInputs() {
        // Higher index tests, check growth, known fib(12)=144
        assertEquals(55, ap01.Series.fibonacci(10));
        assertEquals(89, ap01.Series.fibonacci(11));
        assertEquals(144, ap01.Series.fibonacci(12));
    }

    @Test
    public void testFibonacciEdge() {
        // Test n=1 and n=2 (both should still work)
        assertEquals(1, ap01.Series.fibonacci(1));
        assertEquals(1, ap01.Series.fibonacci(2));
    }
}

File: RepoTransBench/target_projects/Java/C#/AllenDowney_ThinkJavaCode/public_tests/Ap01/SeriesFibonacciEdgePublicTest.cs
using Xunit;
using AllenDowney.ThinkJavaCode;

namespace PublicTests.Ap01
{
    public class SeriesFibonacciEdgePublicTest
    {
        [Fact]
        public void TestFibonacciLow()
        {
            Assert.Equal(13, Series.Fibonacci(7));
            Assert.Equal(21, Series.Fibonacci(8));
        }

        [Fact]
        public void TestFibonacciHighDifferentInputs()
        {
            Assert.Equal(55, Series.Fibonacci(10));
            Assert.Equal(89, Series.Fibonacci(11));
            Assert.Equal(144, Series.Fibonacci(12));
        }

        [Fact]
        public void TestFibonacciEdge()
        {
            Assert.Equal(1, Series.Fibonacci(1));
            Assert.Equal(1, Series.Fibonacci(2));
        }
    }
}